### PR TITLE
Improve search documentation

### LINKED ISSUE #62: Improve search documentation
The search documentations need substantial updates.
- prepare/map-styling-configuration.rst was project-management/configure-search.rst
- fieldwork (field-work/search.rst) 

also see #20
also see #59

### PR DESCRIPTION
**Pull request by lucienicolier**
Improve search documentation
Add animated image to demonstrate search
add document "GoTo Coordinate" search locator
fixes #62

### CODE REVIEW

File: en/fieldwork/search.rst [hunk 1]
```diff
@@ -5,5 +5,13 @@ Tap on the **Search button** on the top-right corner to search in the layers.
 1. Tap on the **name** in the result to pan the map on the feature (the feature will be highlighted in yellow).
 2. Tap on the **attributes button** to open its attributes.
 
-.. image:: ../images/user-guide_search.png
+.. image:: ../images/search.gif
 
+GoTo Coordinate
+---------------
+It is possible to go to Coordinates with the search tool.
```
Comment (m-kuhn) [inline en/fieldwork/search.rst:12]:
```suggestion
It is possible to go to coordinates with the search tool. 
```

File: en/fieldwork/search.rst [hunk 2]
```diff
@@ -5,5 +5,13 @@ Tap on the **Search button** on the top-right corner to search in the layers.
...
 2. Tap on the **attributes button** to open its attributes.
 
-.. image:: ../images/user-guide_search.png
+.. image:: ../images/search.gif
 
+GoTo Coordinate
+---------------
+It is possible to go to Coordinates with the search tool. 
+
+Tap on the **Coordinates** in the result to go to the Coordinates.
```
Comment (m-kuhn) [inline en/fieldwork/search.rst:14]:
```suggestion
Tap on the **coordinates** in the result to go to the coordinates.
```

File: en/fieldwork/search.rst [hunk 3]
```diff
@@ -5,5 +5,13 @@ Tap on the **Search button** on the top-right corner to search in the layers.
 1. Tap on the **name** in the result to pan the map on the feature (the feature will be highlighted in yellow).
 2. Tap on the **attributes button** to open its attributes.
 
-.. image:: ../images/user-guide_search.png
```
Comment (m-kuhn) [inline en/fieldwork/search.rst:8]: If this image is no longer needed, delete it from the repository

**Comment by m-kuhn** [comment]
I wonder if we can get the images with better colors ?!

Do you only have them in this format? Maybe a  webp instead of gif animation could be better?

File: en/prepare/attributes-from.rst [hunk 4]
```diff
@@ -102,6 +102,12 @@ In QField the picture is displayed. By clicking the camera symbol the camera is
 
 On synchronization the sub-directory with the pictures has to be copied as well.
 
+**Warning**
+
+The option "hyperlink" on the attachment widget will disable the camera functionality and enable a hyperlink to open a document (pdf, doc, ...).
+
+ .. figure:: /images/hyperlink_option.PNG
```
Comment (m-kuhn) [inline en/prepare/attributes-from.rst:109]: I think this is important to update in the "external resource" section of the table. It states that only camera is supported.

**Comment by lucienicolier** [comment]
@m-kuhn I had images with better color

File: en/fieldwork/search.rst (hunk 1 above)
Comment (m-kuhn) [reply en/fieldwork/search.rst:12]: @lucienicolier these are not yet resolved (or probably were reverted at some point)

**Comment by lucienicolier** [comment]
Hi @nirvn , is it possible for you to make a review of this PR? It would be nice :) Thanks

**Comment by nirvn** [comment]
@lucienicolier , I think it'd be good to re-do the feature search webp video with a 1.7 build. It look significantly better there.

**Comment by nirvn** [comment]
> Tap on the **Search button** on the top-right corner to search in the layers. 

"Searching in the layers" is probably too vague for newcomers. I'd suggest something along the lines of "Tap on the search button on the top-right corner of the screen to search for features in the current project."

> Tap on the Coordinates in the result to go to the Coordinates. It is always possible to search WGS84 coordinates or current map CRS.

Maybe specify that the go coordinates can be either _longitude,latitude_ or a pair of _x,y_ in the current project coordinate system.

**Comment by lucienicolier** [comment]
@nirvn thanks for your suggestions, I did the text change and will wait for the 1.7 release to re-do the feature search webp.

**Comment by m-kuhn** [comment]
@lucienicolier can you have a look at the unresolved comments above with capitalization of coordinates?

File: en/fieldwork/search.rst [hunk 5]
```diff
@@ -1,9 +1,17 @@
...
+.. image:: ../images/search.webp
 
+GoTo Coordinate
+---------------
+It is possible to go to coordinates with the search tool. 
+
+Tap on the **coordinates** in the result to go to the Coordinates.
```
Comment (m-kuhn) [inline en/fieldwork/search.rst:14]:
```suggestion
Tap on the **coordinates** in the result to go to the coordinates.
```

And one more here ;)

**Pull request merged**